<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteMgrForm</class>
 <widget class="QWidget" name="NoteMgrForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>笔记管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,5,2,2,0,0">
     <item>
      <widget class="QPushButton" name="btnReturn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;&lt;  返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="edtTitle"/>
     </item>
     <item>
      <widget class="QLineEdit" name="edtCrtTime">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtLastTime">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pTitle">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="lwTitles">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pNote">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="NoteTexts"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actAddNote">
   <property name="text">
    <string>添加笔记</string>
   </property>
   <property name="toolTip">
    <string>添加笔记</string>
   </property>
  </action>
  <action name="actDelNote">
   <property name="text">
    <string>删除笔记</string>
   </property>
   <property name="toolTip">
    <string>删除笔记</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
